GoogleMapsApi/Entities/Places/Request/PlacesRequest.cs

```diff
@@ -54,12 +53,11 @@ protected internal override string BaseUrl
 		public RankBy RankBy { get; set; }
 
 		/// <summary>
-		/// Restricts the results to Places matching at least one of the specified types. 
-		/// Types should be separated with a pipe symbol (type1|type2|etc). 
+		/// Restricts the results to Places matching the specified type. 
 		/// See the list of supported types - https://developers.google.com/maps/documentation/places/supported_types
 		/// 
 		/// </summary>
-		public string Types { get; set; }
+		public string Type { get; set; }
```

Comment:
This is the deprication message:

> Notice: The implementation for types in text search requests is changing. The types parameter is deprecated as of February 16, 2016, replaced by a new type parameter which only supports one type per search request. Additionally, the establishment, finance, food, general_contractor, grocery_or_supermarket, health and place_of_worship types will no longer be supported as search parameters (however these types may still be returned in the results of a search). Requests using the deprecated features will be supported until February 16, 2017, after which all text searches must use the new implementation.

So i wouldn't say it's backwards compatible. I'm looking into refactoring now.